openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -268,12 +267,15 @@ def write_project_to_op(project: dict, dbcon: AvalonMongoDB) -> UpdateOne:
         {
             "code": project_code,
             "fps": float(project["fps"]),
-            "resolutionWidth": int(project["resolution"].split("x")[0]),
-            "resolutionHeight": int(project["resolution"].split("x")[1]),
             "zou_id": project["id"],
         }
     )
 
+    proj_res = project["resolution"]
+    if "x" in proj_res:
+        project_data['resolutionWidth'] = int(proj_res.split("x")[0])
+        project_data['resolutionHeight'] = int(proj_res.split("x")[1])
```

Comment:
Would it make sense to at least a log a warning or debug message that resolution data is incorrect?

Reply:
I agree and maybe set resolution to some default (`0`, `0`?).

Reply:
- Should I add a logger instead of all print?
- I would prefer not to set the resolution to zero because in my case the resolution was deleted by mistake, it would be problematic in the software to have a resolution set to 0x0 during artist work.

Reply:
Log would be good here, and I agree that setting to 0 is not a good way here. It would introduce more mistakes potentially